arrows/core/transform_detected_object_set.cxx

```diff
@@ -213,23 +213,24 @@ box_around_box3d( const kwiver::vital::camera_perspective_sptr camera,
     projected_points.row(i) = camera->project(box3d.row(i)).transpose();
   }
 
-  vector_4d out_box;
-  out_box << (vector_2d)projected_points.colwise().minCoeff(),
-             (vector_2d)projected_points.colwise().maxCoeff();
+  auto out_box = vital::bounding_box<double>(
```

Comment:
you could bypass the temporary variable `out_box` and just directly `return vital::bounding_box(`...

Reply:
Aye, that's certainly cleaner.  Fixed here and the other suggested locations as well.